src/io/flutter/inspector/InspectorService.java

```diff
@@ -710,15 +713,20 @@ DiagnosticsNode parseDiagnosticsNodeHelper(JsonElement jsonElement) {
         }));
     }
 
```

Comment:
I accidentally ran the code formatter. All the rest of the changes in this file are whitespace only.